cranelift/codegen/src/isa/aarch64/inst/mod.rs

```diff
@@ -0,0 +1,2515 @@
+//! This module defines aarch64-specific machine instruction types.
+
+#![allow(non_snake_case)]
+#![allow(unused_imports)]
```

Comment:
Could you remove these few remaining `unused_imports` directives in a few files?

Reply:
Done!